module/motion/walk/TorsoMovement/src/TorsoMovement.cpp

```diff
@@ -39,11 +39,6 @@ namespace motion {
             : Reactor(std::move(environment)), subsumptionId(size_t(this) * size_t(this) - size_t(this)) {
 
             on<Configuration>("TorsoMovement.yaml").then([this](const Configuration& config) {
-                double x = config["test"]["x"].as<double>();
```

Comment:
Make sure you also remove it from the config file

Reply:
Does the .yaml need to be deleted entirely?

Reply:
No, just the values in it that you're not using

Reply:
Although, if you don't have a configuration at all you can delete it
It might be better to leave an empty on<Configuration> and yaml file though, for when you want to add some configuration in the future

Reply:
Yeah, it was all just test data.